Sail/src/Sail/entities/systems/Graphics/ParticleSystem.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 #include "..//BaseComponentSystem.h"
 
-class ID3D12GraphicsCommandList4;
```

Comment:
ParticleSystem doesn't have a getByteSize function?

Reply:
The systems which have no data in them does not need to override getByteSize() and add any specific measures.
BaseComponentSystem::getByteSize() will still be called.